### ISSUE TITLE
Possible Error in Patch Size during Tokenization

### ISSUE THREAD

**Issue by JeremyCJM** [Alpha-VLLM/Lumina-mGPT#23]
Should the patch_size here be 16 in Chameleon?
https://github.com/Alpha-VLLM/Lumina-mGPT/blob/104abe453ec1acca5863698629c4db2111b0b3fc/lumina_mgpt/data/item_processor.py#L78

**Comment by ChrisLiu6** [comment]
I agree that this is confusing and maybe I should have used a better naming, but it is not a bug. This patch_size property is not designed to express the downsample ratio of vae, but is instead designed for the image size indicator tokens and serves as the basic unit to measure the size of images. Specifically, if you add the width indicator token id by 1, the width that it actually presents increases by 32 pixel
